assets/css/activitypub-admin.css

```diff
@@ -220,3 +221,24 @@ input.blog-user-identifier {
 .like .dashboard-comment-wrap .comment-author {
 	margin-block: 0;
 }
+
+.activitypub-settings .welcome-tab-close {
+	position: absolute;
+	top: 0px;
+	right: 0px;
+	font-size: 13px;
+	padding: 2px 15px 10px 20px;
+	line-height: 1.23076923;
```

Comment:
```suggestion
	padding: 0 5px 0 20px;
```
How would you feel about this change? It would help with active and focus styles:

Before | After
--- | ---
<img width="834" alt="Screenshot 2025-03-26 at 8 37 38 AM" src="https://github.com/user-attachments/assets/2064bc60-54a8-4d11-9d2c-ab686636afb9" /> | <img width="837" alt="Screenshot 2025-03-26 at 8 37 19 AM" src="https://github.com/user-attachments/assets/b49ae0e4-1455-489e-8a84-e24afa13aaf2" />